moped-database/docker-compose.yml

```diff
@@ -3,7 +3,7 @@
 version: '3.7'
 services:
     hasura:
-        image: hasura/graphql-engine:latest
+        image: hasura/graphql-engine:v2.11.2
```

Comment:
This is a reversal of opinion on my part. I imagined that we would docker pull the graphql-engine image in our local
stack more than I think we do. This will keep our local stacks up to date, and everyone is free to overload this in their checkouts if they want.

Reply:
I like it a lot! I never think to pull the image, as easy as it is to do so